modules/hcp-support-matrix.adoc

```diff
@@ -51,34 +56,50 @@ Ensure a maximum latency of 200 ms between the management cluster and hosted clu
 
 The following table maps {mce-short} versions to the hosted cluster versions that you can create by using the HyperShift Operator that is associated with that version of {mce-short}:
 
+[NOTE]
+====
+Although the HyperShift Operator supports the versions in the following table, {mce-short} supports only as far back as 2 versions earlier than the current version. For example, if the current version is 4.18, {mce-short} supports as far back as version 4.16. If you want to use a version that is earlier than one of the versions that {mce-short} supports, you can detach your hosted clusters from {mce-short} to be unmanaged.
```

Comment:
If you want to use a version that is earlier than one of the versions that {mce-short} supports, you can detach your hosted clusters from {mce-short} to be unmanaged. OR you can use an earlier version of MCE.

Also can we add a sentence or something to tell users to refer to the MCE support matrix?